mycroft/listener/__init__.py

```diff
@@ -272,6 +291,34 @@ def bind(self, stt, fallback_stt=None):
         if fallback_stt:
             self.fallback_stt = fallback_stt
 
+    @property
+    def wakeup_words(self):
+        return {k: v for k, v in self.engines.items()
+                if v.get("wakeup")}
+
+    @property
+    def listen_words(self):
+        return {k: v for k, v in self.engines.items()
+                if v.get("listen")}
+
+    @property
+    def stop_words(self):
+        return {k: v for k, v in self.engines.items()
+                if v.get("stopword")}
+
+    @property
+    def hot_words(self):
+        return {k: v for k, v in self.engines.items()
+                if not v.get("stopword") and not v.get("wakeup")}
+
+    @property
+    def listening_mode(self):
+        return self.responsive_recognizer.listening_mode
+
+    @listening_mode.setter
+    def listening_mode(self, val):
+        self.responsive_recognizer.listening_mode = val
```

Comment:
Should this include a type check?

Reply:
this is only really to make things easier to write down, `self.loop.responsive_recognizer.listening_mode`was huge so this was added only so i could write `self.loop.listening_mode`

it's a proxy property, not really its business to validate, if anything that would be done one layer deeper

Reply:
Ah, right I guess the determination would be made by `responsive_recognizer.listening_mode`